### PR TITLE
Update Remove-DnsServerResourceRecord.md

### PR DESCRIPTION
**Pull request by msbemba**
Added an example to remove SRV records

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1149

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
@msbemba was it your intention to leave the period after '1.1.1.1' in the examples?

File: docset/windows/dnsserver/Remove-DnsServerResourceRecord.md [hunk 1]
```diff
@@ -82,6 +82,13 @@ PS C:\> Remove-DnsServerResourceRecord -ZoneName "contoso.com" -RRType "A" -Name
 
 This command removes the A resource record from a zone named contoso.com that has the name Host01 and the IP address 10.17.1.41.
 
+### Example 4: Remove multiple SRV records
+```
+PS C:\> Remove-DnsServerResourceRecord -RRType SRV -Name "_misc._tcp" -ZoneName "_msdcs.contoso.com" -RecordData "0","10","1234","1.1.1.1."
+```
+
+This command removes all SRV records in the _msdcs.contoso.co zone that have the name _misc._tcp.
```
Comment (get-itips) [inline docset/windows/dnsserver/Remove-DnsServerResourceRecord.md:90]:
```suggestion
This command removes all SRV records in the _msdcs.contoso.com zone that have the name _misc._tcp.
```

**Comment by dariomws** [comment]
> @msbemba was it your intention to leave the period after '1.1.1.1' in the examples?

@tseward It's added automatically when you create the SRV record. It should be there in all the SRV records.

File: docset/winserver2012-ps/dnsserver/Remove-DnsServerResourceRecord.md [hunk 2]
```diff
@@ -65,6 +65,14 @@ PS C:\>Remove-DnsServerResourceRecord -ZoneName "contoso.com" -RRType "A" -Name
 
 This command removes the A resource record from a zone named contoso.com that has the name Host01 and the IP address 10.17.1.41.
 
+### Example 4: Remove multiple SRV records
+```
+PS C:\> Remove-DnsServerResourceRecord -RRType SRV -Name "_misc._tcp" -ZoneName "_msdcs.contoso.com" -RecordData "0","10","1234","1.1.1.1."
+```
+
+This command removes all SRV records in the _msdcs.contoso.co zone that have the name _misc._tcp.
```
Comment (get-itips) [inline docset/winserver2012-ps/dnsserver/Remove-DnsServerResourceRecord.md:73]:
```suggestion
This command removes all SRV records in the _msdcs.contoso.com zone that have the name _misc._tcp.
```

File: docset/winserver2012r2-ps/dnsserver/Remove-DnsServerResourceRecord.md [hunk 3]
```diff
@@ -73,6 +73,13 @@ PS C:\>Remove-DnsServerResourceRecord -ZoneName "contoso.com" -RRType "A" -Name
 
 This command removes the A resource record from a zone named contoso.com that has the name Host01 and the IP address 10.17.1.41.
 
+### Example 4: Remove multiple SRV records
+```
+PS C:\> Remove-DnsServerResourceRecord -RRType SRV -Name "_misc._tcp" -ZoneName "_msdcs.contoso.com" -RecordData "0","10","1234","1.1.1.1."
+```
+
+This command removes all SRV records in the _msdcs.contoso.co zone that have the name _misc._tcp.
```
Comment (get-itips) [inline docset/winserver2012r2-ps/dnsserver/Remove-DnsServerResourceRecord.md:81]:
```suggestion
This command removes all SRV records in the _msdcs.contoso.com zone that have the name _misc._tcp.
```

**Pull request merged**